ArchiSteamFarm/Steam/Integration/ArchiWebHandler.cs

```diff
@@ -369,6 +375,56 @@ public sealed class ArchiWebHandler : IDisposable {
 			return result;
 		}
 
+		[PublicAPI]
+		public async Task<int?> GetPointsBalance() {
+			(bool success, string? accessToken) = await CachedAccessToken.GetValue().ConfigureAwait(false);
+
+			if (!success || string.IsNullOrEmpty(accessToken)) {
+				return null;
+			}
+
+			KeyValue? response = null;
+
+			for (byte i = 0; (i < WebBrowser.MaxTries) && (response == null); i++) {
+				using WebAPI.AsyncInterface loyaltyRewardsService = Bot.SteamConfiguration.GetAsyncWebAPIInterface(LoyaltyRewardsService);
+
+				loyaltyRewardsService.Timeout = WebBrowser.Timeout;
+
+				try {
+					response = await WebLimitRequest(
+						WebAPI.DefaultBaseAddress,
+
+						// ReSharper disable once AccessToDisposedClosure
+						async () => await loyaltyRewardsService.CallAsync(
+							HttpMethod.Get, "GetSummary", args: new Dictionary<string, object>(2, StringComparer.Ordinal) {
+								{ "access_token", accessToken! },
+								{ "steamid", Bot.SteamID.ToString(CultureInfo.InvariantCulture) }
+							}
+						).ConfigureAwait(false)
+					).ConfigureAwait(false);
+				} catch (TaskCanceledException e) {
+					Bot.ArchiLogger.LogGenericDebuggingException(e);
+				} catch (Exception e) {
+					Bot.ArchiLogger.LogGenericWarningException(e);
+				}
+			}
+
+			if (response == null) {
+				Bot.ArchiLogger.LogGenericWarning(string.Format(CultureInfo.CurrentCulture, Strings.ErrorRequestFailedTooManyTimes, WebBrowser.MaxTries));
+				return null;
+			}
+
+			KeyValue pointsInfo = response["summary"]["points"];
+
+			if (pointsInfo == KeyValue.Invalid) {
+				return null;
+			}
+
+			int points = pointsInfo.AsInteger(int.MinValue);
```

Comment:
If I recall correctly, @JustArchi suggested to use `uint` since it appears that negative values are not possible anymore. I don't remember any argument against it, so consider changing type to `uint`.

Reply:
If we go this way then we have to check against `uint.MaxValue` since `0` is expected.

Reply:
We can then just parse `pointsInfo.Value` by ourselves, since `.AsInteger` just wraps up `int.TryParse` call.

Reply:
ASF should consider negative values as invalid because we assume that they should never be returned by Steam.